app/scripts/modules/kubernetes/loadBalancer/details/details.html

```diff
@@ -89,7 +89,9 @@ <h3 select-on-dbl-click>
       <dl ng-if="loadBalancer.service.spec.clusterIP">
         <dt>Cluster IP</dt>
         <dd>
-        {{loadBalancer.service.spec.clusterIP}}
+          <a target="_blank" href="http://{{loadBalancer.service.spec.clusterIP}}">
```

Comment:
Why prefix everything with `http`? Is that necessary? It's not necessarily the case that these endpoints are serving traffic over HTTP